src/main/CommandHandler.cpp

```diff
@@ -542,6 +542,12 @@ CommandHandler::tx(std::string const& params, std::string& retStr)
         decoder::decode_b64(blob, binBlob);
 
         xdr::xdr_from_opaque(binBlob, envelope);
+        if (envelope.type() == ENVELOPE_TYPE_TX_V0)
```

Comment:
in general, it seems we're missing tests for `txNOT_NORMALIZED`.
Envelope test in particular: I am surprised this didn't trip in the multi sig case; maybe we should randomize them better?

We may want to land #2129 before.